pubspec.yaml

```diff
@@ -23,7 +23,7 @@ dependencies:
   sqflite: ^1.3.1
   intl: ^0.16.1
   flutter_pagewise: ^1.2.3
-  cached_network_image: ^2.3.2+1
+  cached_network_image: 
```

Comment:
Please add a version number like before.

Reply:
Ok, actually it was causing an error.

Reply:
Use a version which is compatible

Reply:
Are you using the latest fluttrr version??

Reply:
Changes have been committed :) Tell me if anything remaining.

Reply:
Thanks